<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitReportReporter -->
<testsuite skipped="0" hostname="Tanyas-MacBook-Air.local" name="com.epam.testng.parameterizedtests.PowTest" tests="5" failures="1" timestamp="15 Dec 2017 16:09:16 GMT" time="0.010" errors="0">
  <testcase name="testPower" time="0.001" classname="com.epam.testng.parameterizedtests.PowTest"/>
  <testcase name="testPower" time="0.000" classname="com.epam.testng.parameterizedtests.PowTest"/>
  <testcase name="testPower" time="0.002" classname="com.epam.testng.parameterizedtests.PowTest"/>
  <testcase name="testPower" time="0.002" classname="com.epam.testng.parameterizedtests.PowTest">
    <failure type="java.lang.AssertionError" message="Invalid operation result. Please, try again. expected [276748.42984267586] but found [53981.8756]">
      <![CDATA[java.lang.AssertionError: Invalid operation result. Please, try again. expected [276748.42984267586] but found [53981.8756]at org.testng.Assert.fail(Assert.java:96)at org.testng.Assert.failNotEquals(Assert.java:776)at org.testng.Assert.assertEqualsImpl(Assert.java:137)at org.testng.Assert.assertEquals(Assert.java:118)at com.epam.testng.parameterizedtests.PowTest.testPower(PowTest.java:23)at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)at java.lang.reflect.Method.invoke(Method.java:498)at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:124)at org.testng.internal.Invoker.invokeMethod(Invoker.java:571)at org.testng.internal.Invoker.invokeTestMethod(Invoker.java:707)at org.testng.internal.Invoker.invokeTestMethods(Invoker.java:979)at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:125)at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:109)at java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1149)at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:624)at java.lang.Thread.run(Thread.java:748)]]>
    </failure>
  </testcase> <!-- testPower -->
  <testcase name="testPower" time="0.005" classname="com.epam.testng.parameterizedtests.PowTest"/>
</testsuite> <!-- com.epam.testng.parameterizedtests.PowTest -->
